exercises/dot-dsl/README.md

```diff
@@ -24,21 +24,6 @@ connected by a green line on a yellow background.
 
 Create a DSL similar to the dot language.
 
-## Description of DSL
-
-A graph, in this DSL, is an object of type `Graph`, taking a list of one 
-or more
-
-+ attributes
-+ nodes
-+ edges
-
-described as tuples.
-
-The implementations of `Node` and `Edge` provided in `dot_dsl.py`.
-
-Observe the test cases in `dot_dsl_test.py` to understand the DSL's design.
-
```

Comment:
The this description should still be here! The contents of the `.meta/hints.md` file are put here when someone runs `configlet generate` to regenerate the READMEs.

Reply:
Ok made the change!